### ISSUE TITLE
Datetime Enddate is not type of Datetime

### ISSUE THREAD

**Issue by alexander-schranz** [massiveart-webservices/ZOOLU#164]
set library/sulu/src/Sulu/Events/Event.php:463

``` php
public function setEndDate($endDate)
{
    $this->endDate = $endDate;
    return $this;
}

```

TO

``` php
public function setEndDate($endDate)
{
    if (is_string($endDate)) {
        $this->endDate = new \DateTime($endDate);
    } else {
        $this->endDate = $endDate;
    }
    return $this;
}
```

**Comment by alexander-schranz** [comment]
#189 #188

**Comment by c00n84** [comment]
Please check if it's allready done.

**Comment by alexander-schranz** [comment]
@c00n84 was fixed in #189 #188
